### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Domynis**
- added a readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,153 @@
...
+    ```
+
+2. Install dependencies:
+    ```bash
+    npm install
+    ```
+
+3. Create a `.env` file in the `backend` directory and add your environment variables:
+    ```env
+    DATABASE_URL="postgresql://postgres:postgres@localhost:5432/SporTogether?schema=public"
```
Comment (Tiberiw) [inline README.md]: maybe not necessarily the username and password from your local postgres server.
Consider replacing: DATABASE_URL="postgresql://postgres:postgres@localhost:5432/SporTogether?schema=public"
to : DATABASE_URL="postgresql://[YOUR_SERVER_USERNAME]:[YOUR_SERVER_PASSWORD]@localhost:[YOUR_SERVER_PORT]/[YOUR_DATABASE_NAME]?schema=public"

File: README.md [hunk 2]
```diff
@@ -1,3 +1,153 @@
...
+    PORT=3001
+    SECRET=q3049h8gfhq30n9fa3w90vnh08hN89029329@J!E$
```
Comment (Tiberiw) [inline README.md]: again, not necessarily wrong but the SECRET can be any string, ex: cat or [ANY_SECRET_STRING]. maybe we can keep our SECRET secret 🙂

File: README.md [hunk 3]
```diff
@@ -1,3 +1,153 @@
...
+
+1. Fork the repository.
+2. Create a new branch (`git checkout -b feature/your-feature`).
+3. Commit your changes (`git commit -am 'Add some feature'`).
+4. Push to the branch (`git push origin feature/your-feature`).
+5. Create a new Pull Request.
+
+## Contact
+
+If you have any questions or suggestions, please feel free to contact us at [domibacsro@gmail.com](mailto:domibacsro@gmail.com).
```
Comment (Tiberiw) [inline README.md]: + [amarietiberiu@yahoo.com](mailto:amarietiberiu@yahoo.com) 
( I'm also open for feedback 😁 )

File: README.md [hunk 4]
```diff
@@ -1,3 +1,153 @@
...
+
+---
+
+KipaLogix 2024
```
Comment (Tiberiw) [inline README.md:157]: Very good info page 👏.

**Review by Tiberiw — CHANGES_REQUESTED** [review]

**Review by Tiberiw — APPROVED** [review]

**Review by Bacter16 — APPROVED** [review]

**Review by florinacho337 — APPROVED** [review]

**Pull request merged**